apartment_application_service/utils.py

```diff
@@ -25,3 +25,14 @@ def is_module_available(module_name):
     except ImportError:
         return False
     return True
+
+
+def get_residence_ordering_number(
+    right_of_residence, right_of_residence_is_old_batch=False
+):
```

Comment:
The API for this function could be better. No need for such long argument name and I wouldn't make the boolean parameter optional, because it's very important to get it right.

How about changing the signature to this?
```suggestion
def get_residence_ordering_number(number: int, *, is_old_batch: bool) -> int:
):
```

Reply:
Fixed